### PR TITLE
Update Livy link in README

### PR DESCRIPTION
**Pull request by saulshanabrook**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # sparkmagic
 
-Sparkmagic is a set of tools for interactively working with remote Spark clusters through [Livy](https://github.com/cloudera/hue/tree/master/apps/spark/java), a Spark REST server, in [Jupyter](http://jupyter.org) notebooks.
+Sparkmagic is a set of tools for interactively working with remote Spark clusters through [Livy](http://livy.incubator.apache.org./), a Spark REST server, in [Jupyter](http://jupyter.org) notebooks.
```
Comment (aggFTW) [inline README.md]: Please remove the last dot after org.

Comment (saulshanabrook) [inline README.md]: Fixed

Comment (aggFTW) [inline README.md]: I just realized, can you also make it an https link? Sorry for multiple comments.

Comment (saulshanabrook) [inline README.md]: No worries, fixed. Should I rebase into one commit?

Comment (aggFTW) [inline README.md]: No, it's fine. I can do squash and merge. I'll let the CI build pass and then close.
Thanks for the contribution!

**Pull request merged**